hdr_html_canvas_element.md

```diff
@@ -266,167 +197,85 @@ The component signals are mapped to red, green and blue tristimulus values accor
+* conversion between an HDR and an SDR color space (tone mapping)
 
-* Transfer function: The inverse of the Reference PQ EOTF specified at Rec. ITU-R BT.2100, scaled
+#### Between HDR color spaces
 
-```javascript
-  function connectionTransferFunction(x) {
-    const c1 =  107 / 128;
-    const c2 = 2413 / 128;
-    const c3 = 2392 / 128;
-    const m1 = 1305 / 8192;
-    const m2 = 2523 / 32;
-    const k = 203;
-    if (x < 0) {
-      return 0;
-    } else if (x <= 1) {
-      const p = Math.pow(x, 1 / m2);
-      return (10000 / k) * Math.pow((p - c1) / (c2 - c3 * p), 1 / m1);
-    } else {
-      return (10000 / k);
-    }
-```
+##### General
 
-_Note:_ The factor `k` is selected such that a display luminance of 203 cd/m<sup>2</sup> results in a linear color value of 1 in the connection color space.
+Conversions to and from extended `srgb` are not included since extended `srgb` is trivially converted to and from `srgb-linear`.
 
+Conversions between `rec2100-hlg` and `rec2100-pq` are implicitly specified through conversions to and from `srgb-linear`.
 
-#### Conversions via Color Connection Space
+The domain and range of the conversions consist of all real values.
 
-##### Conversion from extended-sRGB to HLG
+##### `rec2100-hlg` to `srgb-linear`
 
-_Input:_ Full-range non-linear floating-point `extended-srgb` pixel with black at 0.0 and diffuse white at 1.0. Values may exist outside the range 0.0 to 1.0.
+_Input:_ Full-range non-linear floating-point `rec2100-hlg` pixel with black at 0.0 and diffuse white at 0.75. Values may exist outside the range 0.0 to 1.0.
 
-_Output:_ Full-range non-linear floating-point `rec2100-hlg` pixel with black at 0.0 and diffuse white at 0.75. Values may exist outside the range 0.0 to 1.0.
+_Output:_ Full-range non-linear floating-point `srgb-linear` pixel with black at 0.0 and diffuse white at 1.0. Values may exist outside the range 0.0 to 1.0.
 
 _Process:_
- 1. Linearize using the SRGB EOTF
- 2. Convert from `extended-srgb` color space to `rec2100-hlg` color space
- 3. Scale pixel values - See Note 1
- 4. Apply HLG Inverse EOTF to convert to HLG from Pseudo-Display Light with Lw = 302 cd/m2 - See Note 1
-  * apply HLG Inverse OOTF
-  * apply HLG OETF - See Note 2
 
-_Note 1:_ As `rec2100-hlg` is a relative format, the brightness of the virtual monitor used for mathematical transforms can be chosen to be any level. In this transform it is chosen to be 302 cd/m2 so that the brightness of diffuse white matches the diffuse white of sRGB (80 cd/m2). Using the extended range gamma formula in footnote 2 of BT.2100, this also sets the HLG Inverse OOTF to be unity. The value 0.265 is calculated by taking the inverse OETF of 0.75, the `rec2100-hlg` diffuse white level.
-
-_Note 2:_ See section 5.3 in ITU-R BT.2408-4 relating to negative transfer functions in format conversions.
+1. Apply HLG EOTF to convert the non-linear `rec2100-hlg` Signal to linear Pseudo-Display Light with Lw = 302 cd/m2 - See Note 1
```

Comment:
Typo: 302 should be 203

Reply:
I think you are looking at an old version of the document.